dashi/poolqc/pooling_qc_sample.py

```diff
@@ -216,35 +291,87 @@ def update_sampleindices(run):
     return {
         'data': data,
         'layout': {
-            'title': 'Index Clusters per Sample. Passed Samples: %s Threshold: %s' % (
-                samples_passing_clusters, INDEX_THRESHOLD),
-            'xaxis': {'title': 'Sample', 'automargin': True},
+            'title': 'Index Clusters per Sample',
+            'xaxis': {'title': 'Sample', 'automargin': True, 'tickangle': '90', },
             'yaxis': {'title': 'Clusters'},
             'showlegend': False,
+            'barmode': 'group',
 
         }
     }
 
 
+def percent_difference(run, index_threshold):
+    run = run.sort_values('library')
+
+    data = []
+
+    for inx, d in run.groupby('library'):
+        d['Per Cent Difference'] = (d['Clusters'] - index_threshold) / index_threshold * 100
```

Comment:
I get this warning:

```
/home/slazic/Documents/Code/2018-08-27-Dash_trial/dashi/poolqc/pooling_qc_sample.py:310: SettingWithCopyWarning:


A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/indexing.html#indexing-view-versus-copy
```
You are potentially changing the global `df` variable, which would cause big issues if two or more people load this app.